src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -647,7 +647,7 @@ public void remove_rollback_directory_if_exists(string packageName)
 
                 if (installedPackage.Version == availablePackage.Version)
                 {
-                    string logMessage = "{0} v{1} is the latest version available based on your source(s).".format_with(installedPackage.Id, installedPackage.Version);
+                    string logMessage = "{0}{1} v{2} is the latest version available based on your source(s).".format_with(Environment.NewLine, installedPackage.Id, installedPackage.Version);

```

Comment:
This placement was wrong - it would have ended up in the packageresult messages with the extra line as well.